api/src/main/java/com/epam/pipeline/manager/region/CloudRegionManager.java

```diff
@@ -223,6 +238,33 @@ public AbstractCloudRegion loadOrDefault(final Long id) {
                 .orElseGet(this::loadDefaultRegion);
     }
 
+    public void refreshCloudRegionCredKubeSecret() {
+        log.debug("Create Kube secret with cloud region creds if it does not exist.");
+        if (!kubernetesManager.isSecretExist(CP_REGION_CREDS_SECRET)) {
+            log.warn("Secret: " + CP_REGION_CREDS_SECRET + " doesn't exist!");
+            return;
+        }
+        kubernetesManager.refreshSecret(CP_REGION_CREDS_SECRET,
+                loadAll()
+                        .stream()
+                        .filter(region -> region.getProvider().equals(CloudProvider.AZURE))
+                        .map(region -> (AzureRegion) region)
+                        .collect(
+                                Collectors.toMap(azureRegion -> azureRegion.getId().toString(),
+                                        azureRegion ->  {
+                                            final HashMap<String, String> creds = new HashMap<>();
+                                            creds.put(STORAGE_ACCOUNT, azureRegion.getStorageAccount());
+                                            creds.put(STORAGE_KEY, loadCredentials(azureRegion).getStorageAccountKey());
+                                            try {
+                                                return Base64.encodeBase64String(mapper.writeValueAsString(creds).getBytes());
```

Comment:
Maybe extract this code to some util method or class, since such conversion is used several times